api/goldens/aptos_api__tests__transactions_test__test_get_transactions_output_user_transaction_with_script_function_payload.json

```diff
@@ -52,7 +52,7 @@
         "data": {
           "type": "0x1::timestamp::CurrentTimeMicroseconds",
           "data": {
-            "microseconds": "1"
+            "microseconds": "500000"
```

Comment:
Interesting that this changed

Reply:
This is from yanking the context changes from the indexer. I figured it didn't actually matter for the API.